tests/e2e/svcpoliciesandfirewall/svcpoliciesandfirewall_test.go

```diff
@@ -0,0 +1,363 @@
+// This test verifies:
```

Comment:
Note: I hand wrote these corrections, so spacing might be bad, don't just commit them via GH :smile: